src/components/loginSignup/login.jsx

```diff
@@ -89,53 +92,73 @@ export default class Login extends Component {
             <Loader />
           </div>
         ) : (
-            <div className="login">
-              <Form onSubmit={this.submitForm}>
-                <Form.Group>
-                  <Form.Label>Username</Form.Label>
-                  <br></br>
+          <div className="login">
+            <div className="logoDetail">
+              <img
+                src={BackgroundPage}
+                alt="background"
+                className="backgroundImg"
+              />
 
-                  <Form.Control
-                    id="txtbox"
-                    type="text"
-                    placeholder="Enter username"
-                    onChange={this.handleInput('username')}
-                  />
-                  {
-                    <h6 style={{ color: 'red', fontSize: '16px' }}>
-                      {this.state.errors.username}
-                    </h6>
-                  }
-                </Form.Group>
-                <Form.Group>
-                  <Form.Label>Password</Form.Label>
-                  <br></br>
+              <img src={Logo} alt="Logoimage" className="logoImg" />
+            </div>
+
+            <div className="formDetails">
+              <div className="formBox">
+                <div className="signin">Sign In </div>
+                <Form className="form" onSubmit={this.submitForm}>
+                  <Form.Group>
+                    <Form.Label className="label">Username</Form.Label>
+                    <br></br>
 
-                  <Form.Control
-                    id="txtbox"
-                    type="password"
-                    placeholder="Password"
-                    onChange={this.handleInput('password')}
-                  />
-                  {
-                    <h6 style={{ color: 'red', fontSize: '16px' }}>
-                      {this.state.errors.password}
-                    </h6>
-                  }
-                </Form.Group>
-                <Button id="submtBtn" type="submit">
-                  Submit
-              </Button>
-              </Form>
+                    <Form.Control
+                      id="txtbox"
+                      type="text"
+                      placeholder="Enter username"
+                      onChange={this.handleInput('username')}
+                    />
+                    {
+                      <h6 style={{ color: 'red', fontSize: '16px' }}>
+                        {this.state.errors.username}
+                      </h6>
+                    }
+                  </Form.Group>
+                  <Form.Group style={{ marginTop: '10%' }}>
+                    <Form.Label className="label">Password</Form.Label>
 
-              <Button
-                id="signupBtn"
-                type="submit"
-                onClick={this.navigateToSignup}>
-                Signup
-            </Button>
+                    <br></br>
+
+                    <Form.Control
+                      id="txtbox"
```

Comment:
ID clashes with username. Is ID necessary?

Reply:
changed to classname